examples/tools/system/consuming_system_packages.rst

```diff
@@ -0,0 +1,111 @@
+.. _examples_tools_system_consuming_system_packages:
+
+Consuming system requirements only when building a package
+==========================================================
+
+In some cases, you may want to consume system requirements only when building a package, but not when installing it.
+It can be useful when you want to build a package in a CI/CD pipeline, but you don't want to run the system package
+manager when installing the Conan package in a different environment.
+For those cases, there are few approaches that can be used to achieve this goal.
+
+
+Consume a Conan package wrapper for a system package using hidden visibility
+----------------------------------------------------------------------------
+
+In this approach, you can use a Conan recipe to :ref:`wrap the system package<examples_tools_system_package_manager>`
+via the :ref:`system_requirements()<reference_conanfile_methods_system_requirements>` method,
+making it available as a Conan package. Then, the package can be consumed regularly by the method
+:ref:`requirements()<reference_conanfile_methods_requirements>`, but with the parameter ``visible=False``.
+
+.. code-block:: python
+
+    from conan import ConanFile
+
+    class MyPackage(ConanFile):
+        name = "mypackage"
+        settings = "os", "compiler", "build_type", "arch"
+
+        def requirements(self):
+            self.requires("ncurses/system", visible=False)
+
+        ...
+
+This ensures that downstream consumers of the package *mypackage* will not directly invoke the system package manager (e.g., apt-get).
+Only the direct package consumer of the system wrap package for ``ncurses`` will execute the system package manager.
+
+Centralizing and wrapping ``ncurses`` in a single recipe, as mentioned in this example, makes it reusable across multiple
+cases and is good practice to avoid code duplication.
+
+
+Package the system requirement as a header-library or consume only the headers
```

Comment:
This also reads like a derivative or particular case for the other example.

Reply:
Yes, it's a particular case related to the previous example.

Reply:
I think this section is more confusing than beneficial at this point, I'd probably remove it.

Reply:
Removed by the commit [95b4c57](https://github.com/conan-io/docs/pull/3980/commits/95b4c57e6d7733e5080df954f934d50736f4f457)